lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -170,6 +173,7 @@
     CFName(None, 'storm_relative_helicity', 'J kg-1'): G2Param(2, 0, 7, 8),
     CFName('air_potential_temperature', None, 'K'): G2Param(2, 0, 0, 2),
     CFName('air_pressure', None, 'Pa'): G2Param(2, 0, 3, 0),
+    CFName('air_pressure_at_sea_level', None, 'Pa'): G2Param(2, 0, 3, 0),
```

Comment:
This CFName (`air_pressure_at_sea_level`) appears to be associated with two `G2Param`s (on this line and the next line).